pkgs/os-specific/linux/ch9344/default.nix

```diff
@@ -2,16 +2,15 @@
 
 stdenv.mkDerivation rec {
   pname = "ch9344";
-  version = "2.0";
+  version = "2.1";
 
   src = fetchzip {
     name = "CH9344SER_LINUX.zip";
     url = "https://www.wch.cn/downloads/file/386.html#CH9344SER_LINUX.zip";
```

Comment:
Please replace with an archived URL

Reply:
I also independently figured out that their Github is a better source than their release zips. The URL for the release zips stays the same but the content is updated when they have a new release.